<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Through the Mosaic-Glass:</title>
    <meta charset="utf-8" />
    <meta name="author" content="By Sanket Rane" />
    <script src="index_files/header-attrs/header-attrs.js"></script>
    <link rel="stylesheet" href="xaringan-themer.css" type="text/css" />
    <link rel="stylesheet" href="xaringan-custom.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: top, center, title-slide

.title[
# Through the Mosaic-Glass:
]
.subtitle[
## Identifying the Origins of Immune Variability in Our Responses to Diseases
]
.author[
### By Sanket Rane
]

---


count: false










####  Through the looking-glass as a chess game by Glen Downey


&lt;img src="figures/Through_the_Looking-Glass_chess_game.gif" width="25%" style="display: block; margin: auto;" /&gt;

???

- Chess symbolism in alice through the looking glass

- admire chess complexity, well documented by the shanno no.



--

### Chess complexity

- **The Shannon number**: Estimates `\(10^{120}\)` possible games

&gt; Distinct rules of movement for each piece along the 8 `\(\times\)` 8 board &lt;br&gt;
&gt; A diverse set of permissible interactions 


- The state-space complexity of `\(10^{46}\)`

.citeR[
Chinchilkar, ICGA 1996,
Allis, University of Limburg 1994]

???
of a game is the number of legal game positions reachable from the initial position of the game



---

### Complexity and dynamism in the immune system

&lt;hr&gt;


 
#### Multiple subsets with unique properties and complex interactions

 - Naive T and B cells on activation
&gt; Transform into **Effector** cells to curb the infection.  &lt;/br&gt;
&gt; Differentiate into **Memory** cells  and protect against re-exposure.
  

 
--


 &lt;/br&gt;
 
#### Diverse repertoire of clones that identify **antigens** through specific receptors:
  &gt; T cells express T cell receptor (TCR): Diversity &gt; `\(10^{15}\)` &lt;/br&gt;
  &gt; B cells express antibody molecules (Ig): Diversity &gt; `\(10^{14}\)`
  
.citeR[
Mora and Walzack, ArXiv &lt;svg viewBox="0 0 448 512" style="height:1em;position:relative;display:inline-block;top:.1em;" xmlns="http://www.w3.org/2000/svg"&gt;  &lt;g label="icon" id="layer6" groupmode="layer"&gt;    &lt;g transform="matrix(1.5499998,0,0,-1.5499998,-123.19936,553.59966)" id="g4" style="stroke-width:0.645161"&gt;      &lt;path fill="currentColor" d="m 119.65,351.996 c -5.84961,0.13477 -11.1943,-3.29883 -13.502,-8.67578 -2.19629,-5.27051 -0.61914,-8.9668 4.19727,-15.8652 7.05469,-10.3799 78.8242,-96.5625 78.8242,-96.5625 l -15.8828,-14.8633 c -13.3809,-13.3779 -13.9561,-31.333 -1.50977,-43.7754 l 18.4922,-17.6133 -51.5977,-63.377 c -4.00586,-4.26758 -6.48535,-11.7559 -4.24805,-17.1309 2.27832,-5.53613 7.69727,-9.12891 13.6836,-9.07031 3.83398,0.0957 7.43262,1.87402 9.83789,4.86133 l 61.3691,57.0566 94.5762,-90.0703 c 3.19434,-3.08398 7.44336,-4.83984 11.8828,-4.91016 1.60449,0.0039 3.19824,0.245117 4.73242,0.714844 5.7793,1.80566 10.249,6.41797 11.8711,12.252 1.2998,5.47363 -0.27637,11.2334 -4.18164,15.2832 l -83.0859,100.011996 14.8789,13.834 c 11.0957,10.0029 11.1543,27.3906 0.12695,37.4688 l -16.2949,15.6309 56.2559,66.4434 0.0742,0.0859 0.0664,0.0899 c 5.02734,6.53125 7.43164,11.5615 4.83984,17.9395 -3.13379,5.96875 -8.69727,10.29 -15.2559,11.8516 -0.67676,0.0908 -1.35938,0.13574 -2.04297,0.13671 l -0.004,-0.0117 c -4.42871,-0.27051 -8.61035,-2.13086 -11.7754,-5.24023 l -0.13086,-0.10743 -0.12304,-0.11132 -65.207,-59.127 -89.8965,86.2402 c 0,0 -5.33398,6.47754 -10.9707,6.61133 z m 178.104,-33.041 c 0.47852,-0.002 0.95703,-0.0332 1.43164,-0.0957 4.84277,-1.33301 8.95605,-4.54004 11.4316,-8.91016 1.44922,-3.5625 1.00293,-6.45703 -4.19727,-13.2148 l -56.0586,-66.2188 -26.375,25.3027 64.9551,58.9062 c 2.33887,2.41504 5.46484,3.91113 8.8125,4.2168 v 0.0137 z M 192.436,227.402 334.446,57.244 c 2.78711,-3.48926 4.5293,-6.97949 3.3418,-10.9121 -1.16113,-4.30273 -4.43457,-7.71777 -8.68555,-9.05664 -1.08105,-0.333984 -2.20508,-0.503906 -3.33594,-0.505859 -3.20801,0.07617 -6.26953,1.35645 -8.57617,3.58789 l -142.033,135.284999 c -11.1719,11.1719 -9.48242,26.0195 1.39258,36.8945 z" id="path2" style="stroke-width:0.416233"&gt;&lt;/path&gt;    &lt;/g&gt;  &lt;/g&gt;&lt;/svg&gt; 2016,   Yaari &amp; Kleinstein, Genome Med 2015, &lt;br&gt; Davis and Bjorkman, Nature 1988
]

---
class: inverse, center

&lt;br&gt;

&lt;br&gt;
&lt;br&gt;

## The state-space of immune responses is &lt;br&gt; astronomically large

???

- Inside an immune response the possible number of combinations that can arise starting from same initial conditions is pretty large

- most dont start from same init conds




--


&lt;br&gt;
&lt;br&gt;
&lt;br&gt;

## Understand the rules governing the &lt;br&gt; variability in our responses to diseases


???

But we conjecture --

Similar to how a simple set of rules on the chess board can result in an enormous number of possible game positions,

Something similar operates for our immune system
And if such a system of rules exist the we hope to explore and understand them


---

.center[
### Immune variability
]
&lt;hr&gt;

.SimpleBold[
Why do our immune systems produce distinct outcomes in similar disease contexts?
]

???

Immune variability is not a new concept. 
Immunologists have long pondered why our immune systems produce distinct outcomes in similar disease scenarios.


--

&lt;br&gt;


&lt;img src="figures/imm_var1.png" width="90%" style="display: block; margin: auto;" /&gt;

???
For instance, some cancer patients respond readily to immuno-therapy/checkpoint inhibitor blockade while others don't. 
There is variation within and among populations.


---
count:false

.center[
### Immune variability
]
&lt;hr&gt;

.SimpleBold[
Why do our immune systems produce distinct outcomes in similar disease contexts?
]


&lt;br&gt;



&lt;img src="figures/imm_var2.png" width="90%" style="display: block; margin: auto;" /&gt;

???
For example, some infections more prevalent in infants and the elderly than in healthy adults?
There is variation in immune competency across the lifespan within an individual

---
count:false

.center[
### Immune variability
]
&lt;hr&gt;

.SimpleBold[
Why do our immune systems produce distinct outcomes in similar disease contexts?
]
&lt;br&gt;



&lt;img src="figures/imm_var3.png" width="90%" style="display: block; margin: auto;" /&gt;



???
Lastly, for a given **type** of  context the quality of immune response, its strength and how long does the immune protection last varies substantially.
For example, certain vaccines provide lifelong protection, while others require frequent boosters?


.BoldText[Understanding determinants of immune variability remains a fundamental challenge.]


---

### Understanding lymphocyte heterogeneity

&lt;hr&gt;

.citebottom[Doctoral work]

&lt;br&gt;

Dissecting heterogeneity in unimodaly-distributed populations

&lt;br&gt;

.pull-left[

&lt;img src="figures/unimod.png" width="100%" style="display: block; margin: auto;" /&gt;
]

.pull-right[
&lt;br&gt;
&lt;br&gt;

Does the variability in expression of 'population-defining-markers' have any meaning? 
]
&lt;br&gt;

???

We began with a simple question 

When cell populations are selected using a population defining marker -- that is uniquely expressed on them, such as CD4 for CD4 T cells -- does the variability in that proxy marker have any meaning at all? 

---
count:false

### Lymphocyte heterogeneity

&lt;hr&gt;

&lt;br&gt;

Dissecting heterogeneity in unimodaly-distributed populations

&lt;br&gt;

.citebottom[Doctoral work]

.pull-left[

&lt;img src="figures/unimod2.png" width="100%" style="display: block; margin: auto;" /&gt;
]

.pull-right[
&lt;br&gt;
&lt;br&gt;
.SimpleBold[
Is there any difference between the &lt;br&gt;
&lt;span style="color:#638E64;"&gt;High&lt;/span&gt;  and &lt;span style="color:#E37383;"&gt;Low&lt;/span&gt; cells?
]
]

???

Do cells that lie on each ends of the spectrum differ from each other? Or is it all random and they are all stochastically partitioned and keep moving across the distribution?



---

### Heterogeneity in T cells
&lt;hr&gt;

Co-receptor (CD4 and CD8) &lt;span style="color:#E37383;"&gt;Low&lt;/span&gt; naive T cells:


  - Respond poorly to stimulation &amp;mdash;
    - Weaker activation and proliferation than co-receptor &lt;span style="color:#638E64;"&gt;High&lt;/span&gt; cells.
    
    
???
to answer this, I sorted the high and low expressing naive T cells for a bunch of protein markers e.g. CD4, CD8, PD1, TCR, etc. 
We indeed found that for several of these markers the variability in expression meant variability in phenotypic and functional properties of naive T cells. 


--

  - Show a distinct phenotype as compared to co-receptor &lt;span style="color:#638E64;"&gt;High&lt;/span&gt; cells &amp;mdash;
    - Low expression of activation signaling proteins but &amp;uArr; senescent markers
    - **Resemble T cells from old individuals**
    
    
--

&lt;br&gt;

.my-left-column[

&lt;img src="figures/unimod3.png" width="30%" style="display: block; margin: auto;" /&gt;
]

.my-right-column[
Co-receptor &lt;span style="color:#638E64;"&gt;High&lt;/span&gt; &amp;rArr;  &lt;br&gt; Co-receptor &lt;span style="color:#E37383;"&gt;Low&lt;/span&gt; over time &lt;br&gt; **Cellular aging**
]

???
This was back when I was an experimental immunologist and was focusing on naive T cells

To answer this -- I sorted the high and low naive T cells for a bunch of protein markers e.g. CD4, CD8, PD1, TCR, etc. 

For Several of these markers the variability in expression meant variability in phenotype and functional properties of naive T cells. 

Strongest effects  for CD4 and CD8 co-receptors 
- co-receptor low cells showed poor responsiveness as compared to high cells and they also had a distinct phenotype. 
expressed high amounts of senescent markers and essentially resembled T cells from old individuals.


--



&lt;h3 style="font-size:180%;"&gt;
.BoldText[Heterogeneity in T cell responsiveness is linked to the time spent in circulation]
&lt;/h3&gt;

.citeR[
Balyan _et al._ Immunology 2019,
Rane _et al._ BMC Bio 2014
]

???

Over time, majority of co-receptor low cells persisted as low while, high cells lost their CD4/CD8 expression and became low cells, representing the cellular aging process.

We concluded that the Heterogeneity T cell responsiveness is basically linked to the time that they spend in circulation.



---

### Nature versus nurture
&lt;hr&gt;

.center[
.shadedbox2[As cells age &amp;rarr; acquire the 'Low' expression signature &amp;rarr; functionally-impaired]
]

&lt;br&gt;

- Poor T cell responses in old animals can in part be explained by &lt;br&gt;
&amp;#8618;
Enrichment of functionally-impaired cells.

&lt;br&gt;

???

As I was finishing phd I had 2 hypotheses for the selective enrichment of "old" co-receptor Low cells
--


.BoldText[Adaptation model] 

Accrual in T cells' (and their progeny's) fitness as a function of their cell-age
 
   &gt; _e.g.,_ continuous tuning of the threshold for mitogenic stimuli 
   
--

&lt;br&gt;


.BoldText[Selection model] 

Inherent variation in T cells' ability to persist within the circulation
  
  &gt; _e.g.,_ heritable differences in sensitivity to survival signals


???
Alternatively, heterogeneity in lymphocytes  manifest as a result of adaptive changes that cells accrue as they age;

---


### Quantifying lymphocyte ecology

&lt;hr&gt;



#### Multiple processes contribute to **establishment** and **maintenance**.  
&lt;/h3&gt;

 - Division, death, influx from the precursor, onward differentiation, etc.

&lt;br&gt;

&lt;img src="figures/homeostasis_processes.png" width="55%" style="display: block; margin: auto;" /&gt;

&lt;br&gt;


.MyBold[Homeostatic fitness] `\(\Phi = \rho - \delta\)`  
&gt; The ability of a clone (a cell and its progeny) to persist in the population




&lt;!---
class: center


### Ecology&amp;dash;to&amp;dash;response variation
&lt;hr&gt;
&lt;hr&gt;

&lt;br&gt;

&lt;br&gt;

T and B lymphocytes produce clones of varying sizes and disparate cell fates

&lt;br&gt;

&lt;br&gt;

&lt;br&gt;

&lt;img src="figures/response_var.png" width="100%" style="display: block; margin: auto;" /&gt;


&lt;br&gt;

.citeR[
Bresser Nat Imm 2022, Mitchell PNAS 2021, Kibler JEM 2021, Balyan Immunology 2019, Zhou Front Imm 2018, Rane BMC Bio 2014, Gerlach Science 2013, Buccholz Science 2013, Spencer Nature 2009, Hawkins PNAS 2007, Haynes JEM 2005, Tangye Immunity 2004
]

.citebottom[continued...]

---&gt;

---


### Ecology&amp;dash;to&amp;dash;response variation
&lt;hr&gt;

.center[
Variation in cellular states
of the initial responding population &lt;br&gt;
&lt;span style="font-size:180%;"&gt; &amp;#8681; &lt;/span&gt;&lt;br&gt;
Response diversity 
]

???

The high-level idea is 

Variation in cellular-states such as 

- young or old cells 

- particular expression signature

- cells with varying division histories


Of course alternative or null hypothesis is  pure randomness. 

--

&lt;br&gt;
&lt;br&gt;



.blockquote[
&lt;h3 style="color:black;"&gt; &lt;svg viewBox="0 0 352 512" style="position:relative;display:inline-block;top:.1em;fill:#ECB442;height:1em;" xmlns="http://www.w3.org/2000/svg"&gt;  &lt;path d="M96.06 454.35c.01 6.29 1.87 12.45 5.36 17.69l17.09 25.69a31.99 31.99 0 0 0 26.64 14.28h61.71a31.99 31.99 0 0 0 26.64-14.28l17.09-25.69a31.989 31.989 0 0 0 5.36-17.69l.04-38.35H96.01l.05 38.35zM0 176c0 44.37 16.45 84.85 43.56 115.78 16.52 18.85 42.36 58.23 52.21 91.45.04.26.07.52.11.78h160.24c.04-.26.07-.51.11-.78 9.85-33.22 35.69-72.6 52.21-91.45C335.55 260.85 352 220.37 352 176 352 78.61 272.91-.3 175.45 0 73.44.31 0 82.97 0 176zm176-80c-44.11 0-80 35.89-80 80 0 8.84-7.16 16-16 16s-16-7.16-16-16c0-61.76 50.24-112 112-112 8.84 0 16 7.16 16 16s-7.16 16-16 16z"&gt;&lt;/path&gt;&lt;/svg&gt; &amp;nbsp; &amp;nbsp; If we know the mechanisms that regulate lymphocyte populations
&lt;/h3&gt;

 &gt; ### .BoldText[Then we can conjecture the cell-state distribution of the initial responders]

]





---

### Mechanisms of lymphocyte maintenance

&lt;hr&gt;

How are lymphocyte populations regulated at steady-states?


.put-bottom-center[
&lt;img src="figures/fitness_col.png" width="40%" style="display: block; margin: auto;" /&gt;

Fitness: Net-effect of birth and death processes
]

.pull-left[

&lt;img src="figures/neutral_model.png" width="100%" style="display: block; margin: auto;" /&gt;
]

--

.pull-right[

&lt;img src="figures/competition_model.png" width="100%" style="display: block; margin: auto;" /&gt;
]




---

### Mechanisms of lymphocyte maintenance

&lt;hr&gt;


How are lymphocyte populations regulated at steady-states?


.put-bottom-center[
&lt;img src="figures/fitness_col.png" width="40%" style="display: block; margin: auto;" /&gt;

Fitness: Net-effect of birth and death processes
]

.pull-left[

&lt;img src="figures/adaptation_model.png" width="100%" style="display: block; margin: auto;" /&gt;
]

--

.pull-right[

&lt;img src="figures/selection_model.png" width="100%" style="display: block; margin: auto;" /&gt;
]



---
### Towards unifying models of lymphocyte homeostasis
&lt;hr&gt;


&lt;img src="figures/towards_m.png" width="95%" style="display: block; margin: auto;" /&gt;


.citebottom[
Babushku _et al._ Nature Comm. 2024, Rane _et al._ eLife 2022; &lt;br&gt; 
Verheijn _et al._ Cell Reports 2020; Mold _et al._ PLoS Bio 2019; Rane _et al._ PLoS Bio 2018
]







---



.center[
### Cell-fate mapping using bone marrow transplantation system
]

&lt;hr&gt;


&lt;img src="figures/Buchi_strategy.png" width="70%" style="display: block; margin: auto;" /&gt;

&lt;br&gt;

Busulfan treatment:
&gt; Eliminates hematopoietic stem cells (HSC) in bone marrow

&gt; Leaves peripheral lymphoid compartments intact

.right[
#####  Seddon lab UCL
]


&lt;!---
class: center

### Total numbers of T and B cell subsets
&lt;hr&gt;

&lt;img src="figures/buchi_stability.png" width="55%" style="display: block; margin: auto;" /&gt;

&lt;br&gt;
**Stable chimerism with no/minimal effects on peripheral lymphocyte compartments**
---&gt;

---
class: center, top

### Donor invasion kinetics
&lt;hr&gt;


.right-column[
&lt;img src="figures/buchi_donorfraction.png" width="70%" style="display: block; margin: auto;" /&gt;
]


.left-column[
&lt;br&gt;
CD4 T cell subset as an example
]

---

### The kinetics of donor-percolation are rich in information

&lt;hr&gt;


&lt;img src="figures/Nfd_schema.png" width="100%" style="display: block; margin: auto;" /&gt;

&lt;br&gt; 
.BoldText[Set point –]

- Whether replacement is complete or incomplete

&gt; Heterogeneity within the compartment

--

.BoldText[Upslope –]

- How fast host cells are replaced by donor-BM-derived cells

&gt; Interplay between influx and turnover

---

### Connecting biology to theory

&lt;hr&gt;

&lt;br&gt;

&lt;img src="figures/bio_math.png" width="80%" style="display: block; margin: auto;" /&gt;


---

class: center
### Data from Busulfan chimeras

&lt;hr&gt;

Different colors denote groups of animals with different ages at BMT

&lt;br&gt;

&lt;img src="figures/cd4_data.png" width="75%" style="display: block; margin: auto;" /&gt;

--


&lt;img src="figures/cd8_data.png" width="75%" style="display: block; margin: auto;" /&gt;

---
## Model validation using Bayesian approach
&lt;hr&gt;


&gt; At each time point `\(i\)` (&gt;= 1 animal/s), 

&gt; `\(j\)` quantities such as total numbers and `\(f_d\)`

&lt;br&gt;

$$
`\begin{aligned}
&amp;y_{ij} \sim \text{Normal}(\mu_{ij}, \Sigma) \quad \quad \quad \quad \quad \quad  &amp;[\text{Likelihood}] \\
&amp;\mu_i = M_j(\text{time}_i, \theta)  &amp;[\text{Model prediction}]\\
\end{aligned}`
$$
.center[
`\(\theta:\)` Vector of model parameters and 
`\(\Sigma:\)` Covariance matrix
]


&lt;br&gt;

.SimpleBold[Output:]
&gt; Posterior distribution of model parameters `\(P(\theta | y)\)`  



&gt; Estimate of confidence in the model &amp;rArr; **Model weight** 

???
We define the likelihood of data y given the model.

which is the probability density of data across the parameter distribution by assuming that the errors are normally distributed. 

Normalized product of probability densities of both datasets and the prior distribution of params.

- This probability density is then used to update our prior beliefs in model parameters using the bayesian approach.


Parameters are sampled from the joint **prior** density using **Hamiltonian Monte Carlo** algorithm.

???
We measure this  out of sample prediction error using Leave-One-Out cross validation, in which a single observation is treated as test dataset and rest of the observations are trated as training data. The process is repeated for all 'n' observations and the prediction error for each left-out test set is aggregated to calculate Leave-One-Out information criterion (LOOIC).

???
We then used the estimated LOO-IC values to assess the relative support for models using the analog of the Akaike weight which gives us the probability that a given model will explain new data better than other models considered in this analysis.
---
class: center
### Life-expectancy of naive T cells increases with cell-age

&lt;hr&gt;

The adaptation model received strongest statistical support: .BoldText[&amp;gt; 80% Model weight]

&lt;br&gt;

&lt;img src="figures/cd4_res.png" width="75%" style="display: block; margin: auto;" /&gt;



&lt;img src="figures/cd8_res.png" width="75%" style="display: block; margin: auto;" /&gt;


---

### Mechanistic meta analysis
&lt;hr&gt;



.citebottom[
Rane _et al._ eLife 2021, Mold _et al._ Plos Biology 2019, Rane _et al._ Plos Biology 2018
]


&lt;img src="figures/Tcell_metatable2.png" width="90%" style="display: block; margin: auto;" /&gt;

&lt;br&gt;
Across multiple data sets derived from diverse experiments:

- The adaptation model provides the most parsimonious explanation of naive T cell maintenance.

--

.shadedbox2[
Can waning T cell mediated immunity in the elderly be explained by the accumulation of old and functionally impaired T cells?  
]

---
    
&lt;!---

.my-left-column[
&lt;img src="figures/mold_paper.png" width="80%" style="display: block; margin: auto;" /&gt;
]



.my-right-column[

&lt;img src="figures/mold_paper2.png" width="80%" style="display: block; margin: auto;" /&gt;
]

---&gt;



---




class:center

### Modeling fate-decisions and response-dynamics in the B cell lineage
&lt;hr&gt;

&lt;br&gt;

&lt;img src="figures/Bcell_project.png" width="100%" style="display: block; margin: auto;" /&gt;


---



## B cell maturation in the spleen
&lt;hr&gt;

- B cells transition through **short-lived** developmental stages in the spleen.


- Late-stage transitional (T2) B cells bifurcate into fully mature Follicular (Fo) or Marginal Zone (MZ) B cells.

&lt;br&gt;


&lt;img src="figures/bcell_matu.jpg" width="55%" style="display: block; margin: auto;" /&gt;

---

## Division of labour and space
&lt;hr&gt;


Marginal Zone is an anatomical barrier that divides lymphocyte-rich follicles and the erythrocyte-rich red pulp.

.pull-left[
&lt;img src="figures/MZ_viz1.jpg" width="100%" style="display: block; margin: auto;" /&gt;

.citeR[*Zimber-Strobl Lab* &lt;br&gt;
Lechner _et al.,_ Nat Comm 2021 
]

]

.pull-right[



- **MZ B cells** are strategically positioned to identify and respond to **blood-borne** antigens.

  .BoldText[Gatekeepers of the immune system]
  
  
&lt;br&gt;

- **Fo B cells** typically respond to antigens presented by Dendritic cells and Macrophages in a T cell dependent manner and produce high-affinity antibodies.
  


]


???
5 % of circulation goes through spleen
blood cicrulation opens into marginal sinus where blood slows down so that Mphages and B cells guarding the sinus can filter it for antigens/pathogens.


---

### Division history of B cells
&lt;hr&gt;

- We used the **busulfan bone marrow** transplantation system to measure _total numbers_ and `\(f_d\)` in Fo and MZ B cells.


- In addition, we measured Ki67 expression &amp;rArr; extent of cell division in lymphocytes.




&lt;br&gt;

.pull-left[
- T and B cells continue expressing Ki67 protein for &amp;sim; 4 days after the division.


&gt; Residual expression from previous developmental stage? &lt;br&gt;
 &lt;br&gt;
&gt; Division in the current stage?

.citeR[
Verheijn _et al._ Cell Reports 2020
]
]

.pull-right[
&lt;img src="figures/div_hist.png" width="80%" style="display: block; margin: auto;" /&gt;


]

---

class: center
### Neutral model explains MZ B cell dynamics
&lt;hr&gt;

.middle[
&lt;img src="figures/MZ_fits2.png" width="85%" style="display: block; margin: auto;" /&gt;
]


---

class: center

### Time (Host age) dependence in FO B cell dynamics
&lt;hr&gt;

.middle[
&lt;img src="figures/FM_results2.png" width="85%" style="display: block; margin: auto;" /&gt;
]


---
count:false

### Insights into naive B cell homeostasis
&lt;hr&gt;


#### Naive B cells divide **very rarely.**
  - Inter-division times Fo B cells &amp;approx;  &gt; 1 year (CI: 0.6-23)
  - Inter-division times MZ B cells &amp;approx;  6 months (CI: 4-7)
  
  
--

#### Life-expectancy  of Fo B cells increases with mouse age.
  - Mean lifespan in a
    &gt; 10 wk old mouse &amp;approx;  35 days (CI: 2.5-4) &lt;/br&gt;
    &gt; 2 year old mouse &amp;approx;  61 days


#### Mean residence time of MZ B cells &amp;approx; 64 days (CI: 54-72) &lt;/br&gt;
    


#### Constitutive replenishment
  - Approximately ** 3% of Fo and  1.2% of MZ B pool** is replaced daily by source influx.
  

.citeR[
Unpublished data and Verheijn _et. al._ Cell Reports 2020
]


---


class:center

### Immune-response dynamics of B lymphocytes

&lt;hr&gt;
&lt;br&gt;


&lt;img src="figures/bcell_gc.png" width="80%" style="display: block; margin: auto;" /&gt;


---
count:false
class:center

### Immune-response dynamics of B lymphocytes

&lt;hr&gt;
&lt;br&gt;


&lt;img src="figures/bcell_gc2.png" width="80%" style="display: block; margin: auto;" /&gt;



De novo (Notch2-dependent) generation of MZ B cells during an immune response

.citebottom[
Babushku _et al._ Nature Communications 2024
]




???

### .BoldText[Does immune activation modulate B cell differentiation pathways?]



---

### Tracking antigen-specific B cells
&lt;hr&gt;


**C&lt;sub&gt;&amp;gamma;1&lt;/sub&gt;Cre reporter mice** immunized with NP-CGG antigen

- Responding B cells permanently express the reporter protein.

&lt;br&gt;

&lt;img src="figures/imm_data.png" width="80%" style="display: block; margin: auto;" /&gt;

--


- Notch2 KO: Responding B cells express reporter and delete Notch2

- Notch2 is required for differentiation into MZ B phenotype &lt;br&gt;
&amp;#8618; Notch2 KO data helps modeling

---

### Dynamical modelling of antigen-specific B cell response
&lt;hr&gt;


.pull-left[
&lt;img src="figures/branched_imm.png" width="52%" style="display: block; margin: auto;" /&gt;
]

.pull-right[
&lt;img src="figures/linear_imm.png" width="55%" style="display: block; margin: auto;" /&gt;
]

--
&lt;br&gt;

&lt;img src="figures/imm_fit.png" width="100%" style="display: block; margin: auto;" /&gt;


---


### What does this mean?

&lt;hr&gt;

#### Two likely (but not mutually exclusive)  theories: 

 - Antigen-specific MZ B cell clones (or at least a large fraction of them) are
 
  - Delegated to the marginal zone to improve protective coverage and prevent a concurrent systemic breach
  
--
  
  - Archived as memory-like clones for protection against future infections


--

&lt;br&gt;



#### A novel differentiation pathway or cell-state during B cell responses.
&gt; Notch2 dependent: &lt;br&gt;
&gt; Potential target for modulating the breadth and intensity of antibody responses.

&lt;svg viewBox="0 0 512 512" style="height:1em;position:relative;display:inline-block;top:.1em;" xmlns="http://www.w3.org/2000/svg"&gt;  &lt;path d="M201.5 174.8l55.7 55.8c3.1 3.1 3.1 8.2 0 11.3l-11.3 11.3c-3.1 3.1-8.2 3.1-11.3 0l-55.7-55.8-45.3 45.3 55.8 55.8c3.1 3.1 3.1 8.2 0 11.3l-11.3 11.3c-3.1 3.1-8.2 3.1-11.3 0L111 265.2l-26.4 26.4c-17.3 17.3-25.6 41.1-23 65.4l7.1 63.6L2.3 487c-3.1 3.1-3.1 8.2 0 11.3l11.3 11.3c3.1 3.1 8.2 3.1 11.3 0l66.3-66.3 63.6 7.1c23.9 2.6 47.9-5.4 65.4-23l181.9-181.9-135.7-135.7-64.9 65zm308.2-93.3L430.5 2.3c-3.1-3.1-8.2-3.1-11.3 0l-11.3 11.3c-3.1 3.1-3.1 8.2 0 11.3l28.3 28.3-45.3 45.3-56.6-56.6-17-17c-3.1-3.1-8.2-3.1-11.3 0l-33.9 33.9c-3.1 3.1-3.1 8.2 0 11.3l17 17L424.8 223l17 17c3.1 3.1 8.2 3.1 11.3 0l33.9-34c3.1-3.1 3.1-8.2 0-11.3l-73.5-73.5 45.3-45.3 28.3 28.3c3.1 3.1 8.2 3.1 11.3 0l11.3-11.3c3.1-3.2 3.1-8.2 0-11.4z"&gt;&lt;/path&gt;&lt;/svg&gt; &amp;nbsp; Boosting vaccine efficacy or therapeutic interventions for autoimmune pathologies. 

???
We can leverage this information to design 

vaccine improvement? autoimmune?

---

### Questions to explore

&lt;hr&gt;

#### Notch2 guides the differentiation of activated B cells  to MZ fate:

&gt; Is there any interaction between Notch2 and BCR?

&gt; What role does BCR affinity play in MZ versus GC fate-determination?

--


&lt;br&gt;

#### To what extent is the MZ B cell generation at steady-states driven by (self) antigen activation?

&gt; Malignant transformation from chronic or unwarranted activation

&gt; Mouse models to study MZ B cell lymphomas


.right[
&lt;h3 style="font-size:150%;"&gt;Apoorva Singh &amp;nbsp;&amp;nbsp; &lt;/h3&gt;
]
.right-bottom-fixed[
&lt;img src="figures/apoorva_pic.jpeg" width="40%" style="display: block; margin: auto;" /&gt;
]


???

Chronic or unwarranted activation linked with malignant transformation?


---


### Lineage tracing to pinpoint cell-states
&lt;hr&gt;


We are interested in understanding how distinct cell-states
 &gt; Emerge during immune responses &lt;br&gt;
 &gt; Evolve over time

--

&lt;br&gt;

.my-left-column[

- Functional diversification:
&gt; Differentiation to Plasma, memory B cells, etc.

- Affinity maturation:
 &gt; B cells undergo somatic hypermutation (SHM) and selection events 
 
 &lt;br&gt;
 
 .BoldText[
Both occur concurrently in the GC phase
]
]


.my-right-column[
&lt;br&gt;

&lt;img src="figures/concurrent_b.png" width="90%" style="display: block; margin: auto;" /&gt;

]

--


#### .SimpleBold[&lt;svg viewBox="0 0 512 512" style="height:1em;position:relative;display:inline-block;top:.1em;" xmlns="http://www.w3.org/2000/svg"&gt;  &lt;path d="M444.52 3.52L28.74 195.42c-47.97 22.39-31.98 92.75 19.19 92.75h175.91v175.91c0 51.17 70.36 67.17 92.75 19.19l191.9-415.78c15.99-38.39-25.59-79.97-63.97-63.97z"&gt;&lt;/path&gt;&lt;/svg&gt; &amp;nbsp; Precursor-progeny relationships between B cell subsets based on SHM patterns]

---

### Single-cell BCR and RNA sequencing
&lt;hr&gt;


.citebottom[
In collaboration with **Schimdt-Supprian Lab**, TUM and **Zimber-Strobl Lab**, Helmholtz Munich
]

.pull-left[

&lt;img src="figures/response_kinetic.png" width="75%" style="display: block; margin: auto;" /&gt;
]

.pull-right[
- Isolate antigen-specific B cells at three time points.
&gt; B1-8&lt;sup&gt;lo&lt;/sup&gt; BCR transgenics 

{{content}}

]


--

&lt;img src="figures/clone_tree.png" width="75%" style="display: block; margin: auto;" /&gt;


???

We will generate clone-trees and evaluate how often clones from the same branch end up in the same subset... for example.


&lt;br&gt;

Combining BCR and RNA expression dynamics 

- Identify gene signatures that coincide with excessive division or preferential selection &lt;br&gt;
 &amp;#8618;
 Pinpoint cell-states that are most permissive to malignant transformation.


---


### Models representing continuous transition of cellular states
&lt;hr&gt;


&lt;svg viewBox="0 0 352 512" style="position:relative;display:inline-block;top:.1em;fill:#ECB442;height:1.5em;" xmlns="http://www.w3.org/2000/svg"&gt;  &lt;path d="M96.06 454.35c.01 6.29 1.87 12.45 5.36 17.69l17.09 25.69a31.99 31.99 0 0 0 26.64 14.28h61.71a31.99 31.99 0 0 0 26.64-14.28l17.09-25.69a31.989 31.989 0 0 0 5.36-17.69l.04-38.35H96.01l.05 38.35zM0 176c0 44.37 16.45 84.85 43.56 115.78 16.52 18.85 42.36 58.23 52.21 91.45.04.26.07.52.11.78h160.24c.04-.26.07-.51.11-.78 9.85-33.22 35.69-72.6 52.21-91.45C335.55 260.85 352 220.37 352 176 352 78.61 272.91-.3 175.45 0 73.44.31 0 82.97 0 176zm176-80c-44.11 0-80 35.89-80 80 0 8.84-7.16 16-16 16s-16-7.16-16-16c0-61.76 50.24-112 112-112 8.84 0 16 7.16 16 16s-7.16 16-16 16z"&gt;&lt;/path&gt;&lt;/svg&gt; &amp;nbsp; 
Dynamical model to map birth-death-selection events in individual clones

--

- Generate phylogeny-based simulations from the model

--

- Map the simulations on the observed data



- Infer parameters regulating clonal dynamics

--

&lt;br&gt;

&lt;svg viewBox="0 0 352 512" style="position:relative;display:inline-block;top:.1em;fill:#ECB442;height:1.5em;" xmlns="http://www.w3.org/2000/svg"&gt;  &lt;path d="M96.06 454.35c.01 6.29 1.87 12.45 5.36 17.69l17.09 25.69a31.99 31.99 0 0 0 26.64 14.28h61.71a31.99 31.99 0 0 0 26.64-14.28l17.09-25.69a31.989 31.989 0 0 0 5.36-17.69l.04-38.35H96.01l.05 38.35zM0 176c0 44.37 16.45 84.85 43.56 115.78 16.52 18.85 42.36 58.23 52.21 91.45.04.26.07.52.11.78h160.24c.04-.26.07-.51.11-.78 9.85-33.22 35.69-72.6 52.21-91.45C335.55 260.85 352 220.37 352 176 352 78.61 272.91-.3 175.45 0 73.44.31 0 82.97 0 176zm176-80c-44.11 0-80 35.89-80 80 0 8.84-7.16 16-16 16s-16-7.16-16-16c0-61.76 50.24-112 112-112 8.84 0 16 7.16 16 16s-7.16 16-16 16z"&gt;&lt;/path&gt;&lt;/svg&gt; &amp;nbsp; 
Identify gene signatures that coincide with excessive division or preferential selection &lt;br&gt;

&gt; Pinpoint cell-states that are most permissive to malignant transformation


---

### &lt;span style="color:#9c1c44"&gt;Mo&lt;/span&gt;dular &lt;span style="color:#9c1c44"&gt;S&lt;/span&gt;tochastic and &lt;span style="color:#9c1c44"&gt;A&lt;/span&gt;daptive &lt;span style="color:#9c1c44"&gt;I&lt;/span&gt;terating &lt;span style="color:#9c1c44"&gt;C&lt;/span&gt;lusters

&lt;hr&gt;

The **Mosaic model** is a probabilistic mapping method to track population-structures &lt;br&gt; across data points


.pull-left[

&lt;br&gt;

&lt;img src="figures/mosaic_pre.png" width="75%" style="display: block; margin: auto;" /&gt;
]


.pull-right[

- Divide the state-space into **classes**


{{content}}

]

--

- Model evolves from the initial distribution stochastically


{{content}}

--

- Classes &amp;mdash;

 - Pre-defined:  BCR clones, age-distribution, etc.

 - Learned from the data: Expression signature, Treated or Untreated groups, etc.



---
### Mosaic model operation
&lt;hr&gt;


Example: Mapping BCR clonal dynamics


&lt;img src="figures/mosaic_post0.png" width="35%" style="display: block; margin: auto;" /&gt;

--


&lt;img src="figures/mosaic_post.png" width="80%" style="display: block; margin: auto;" /&gt;

---

### Prediction of the dynamics of activation-induced MZ B clones
&lt;hr&gt;

- Using parameter estimates derived from dynamical modeling of B cell response


- Initial conditions: Uniformly distributed 100 Mosaic classes (_i.e._ 100 clones)
 - Each with a single member

&lt;br&gt;

&lt;img src="figures/mosaic_preds.png" width="80%" style="display: block; margin: auto;" /&gt;

.citeR[
Colored curves denote dynamics of individual clones
]



---
class: center, middle

&lt;img src="figures/ind_clones0.png" width="100%" style="display: block; margin: auto;" /&gt;

---
class: center, middle
count:false

&lt;img src="figures/ind_clones1.png" width="100%" style="display: block; margin: auto;" /&gt;

.BoldText[
Clustering based on the dynamical properties
]






---



### Bayesian inverse problem
&lt;hr&gt;

What is an inverse problem?

&gt; Given the observed data `\(x\)` :

&gt; Infer the **causal factors (model parameters `\(\theta\)`)** that generate the data

--

&lt;br&gt;


&lt;img src="figures/pseudocode.png" width="65%" style="display: block; margin: auto;" /&gt;
---

### Need for a new approach 
&lt;hr&gt;

- Curse of dimensionality: 
&gt; Sampling from high-dimensional parameter space &lt;br&gt;  &amp;#8658; **Computationally expensive**

--

- Intractable likelihood:
&gt;  In case of multidimensional data (like ours) &lt;br&gt; &amp;#8658; **Likelihoods are oftentimes ill-defined**


--

&lt;br&gt;

.BoldText[
Question of interest:
]
Is there a way to utilize the modern deep learning architectures to process high-dimensional, sequential data for Bayesian inverse problems?

---

### Autoencoder 
&lt;hr&gt;

- Encoder: Compress input into a latent-space of (smaller) dimension

- Decoder: Reconstruct input from the latent-space

&lt;br&gt;

&lt;img src="figures/autoencoder_schema.png" width="55%" style="display: block; margin: auto;" /&gt;

.citeR[
Towards Data Science
]

--

&lt;br&gt;

.center[
Typical usage &amp;mdash; Denoising, Recoloring, Watermark-removal, etc.
]


---

### Variational Auto-Encoder (VAE) 
&lt;hr&gt;

Autoencoders are deterministic and lack generative capabilities.

Key idea: Construct a distribution (in the latent space)

&gt; The latent variables `\(z\)` are drawn from a probability distribution depending on the input `\(x\)`

&gt; The output `\(\tilde{x}\)` is reconstructed probabilistically from `\(z\)`

&lt;br&gt;


&lt;img src="figures/vae_3.png" width="70%" style="display: block; margin: auto;" /&gt;

.citeR[
Towards Data Science
]


---


### RNN-VAE 
&lt;hr&gt;

Key idea: Combine Recurrent Neural Network (RNN) with VAE 

- To process sequential data

    - Time-series or time-varying observations
    
    - Data points from multiple individuals

--

&lt;br&gt;


- To generate latent compact representation of the multidimensional data that
    
    - Preserves local information while reducing dimensionality 
    
    - Learns dependencies between successive data points
    
--
    
    - For a mathematical model representing the data:

      &gt; Identifies the model parameters (_&amp;theta;_)

---

### Proposed approach

&lt;hr&gt;

.BoldText[Goal]

&gt; Infer causality: Estimate model parameters


&lt;br&gt;
&lt;br&gt;


&lt;img src="figures/vae_model.png" width="90%" style="display: block; margin: auto;" /&gt;


&lt;br&gt;


---
count:false

### Proposed approach

&lt;hr&gt;

.BoldText[Goal]

&gt; Infer causality: Estimate model parameters


&lt;br&gt;
&lt;br&gt;

&lt;img src="figures/vae_model2.png" width="70%" style="display: block; margin: auto;" /&gt;


.right[
&lt;h3 style="font-size:150%;"&gt;Jun Won Park &amp;nbsp;&amp;nbsp; &lt;/h3&gt;
]

.right-bottom-fixed[

&lt;img src="figures/Jun_pic.png" width="28%" style="display: block; margin: auto;" /&gt;
]

---

### Lotka-Volterra predator-prey system
&lt;hr&gt;

&lt;br&gt;

.left-column[

&lt;br&gt;&lt;br&gt;&lt;br&gt;


$$
\frac{dx}{dt} = \alpha \, x - \beta \, x \, y 
$$
$$
\frac{dy}{dt} = \delta \, x \, y - \gamma \, y 
$$
]
.right-column[
&lt;img src="figures/lotka_volterra.png" width="100%" style="display: block; margin: auto;" /&gt;
]

---

### Lotka-Volterra predator-prey system
&lt;hr&gt;


&lt;img src="figures/LV_data.png" width="80%" style="display: block; margin: auto;" /&gt;


---


### Results: Lotka-Volterra predator-prey system
&lt;hr&gt;

&lt;img src="figures/LV_pred.png" width="80%" style="display: block; margin: auto;" /&gt;


.citebottom[
We treated _&amp;alpha;_ and _&amp;delta;_ as free parameters; &lt;br&gt;
_&amp;beta;_ and _&amp;gamma;_ were fixed.
]

---

### Results: Lotka-Volterra predator-prey system

&lt;hr&gt;

&lt;img src="figures/LV_post.png" width="90%" style="display: block; margin: auto;" /&gt;



---

### Benchmarking

&lt;hr&gt;

&lt;br&gt;

&lt;img src="figures/barplot.png" width="100%" style="display: block; margin: auto;" /&gt;


&lt;br&gt;

.citeR[
Best performance: `\(E(\cdot)\)`  closest to true values (1, 1) and lowest Variance &lt;br&gt;
Dinh _et al.,_ ArXiv &lt;svg viewBox="0 0 448 512" style="height:1em;position:relative;display:inline-block;top:.1em;" xmlns="http://www.w3.org/2000/svg"&gt;  &lt;g label="icon" id="layer6" groupmode="layer"&gt;    &lt;g transform="matrix(1.5499998,0,0,-1.5499998,-123.19936,553.59966)" id="g4" style="stroke-width:0.645161"&gt;      &lt;path fill="currentColor" d="m 119.65,351.996 c -5.84961,0.13477 -11.1943,-3.29883 -13.502,-8.67578 -2.19629,-5.27051 -0.61914,-8.9668 4.19727,-15.8652 7.05469,-10.3799 78.8242,-96.5625 78.8242,-96.5625 l -15.8828,-14.8633 c -13.3809,-13.3779 -13.9561,-31.333 -1.50977,-43.7754 l 18.4922,-17.6133 -51.5977,-63.377 c -4.00586,-4.26758 -6.48535,-11.7559 -4.24805,-17.1309 2.27832,-5.53613 7.69727,-9.12891 13.6836,-9.07031 3.83398,0.0957 7.43262,1.87402 9.83789,4.86133 l 61.3691,57.0566 94.5762,-90.0703 c 3.19434,-3.08398 7.44336,-4.83984 11.8828,-4.91016 1.60449,0.0039 3.19824,0.245117 4.73242,0.714844 5.7793,1.80566 10.249,6.41797 11.8711,12.252 1.2998,5.47363 -0.27637,11.2334 -4.18164,15.2832 l -83.0859,100.011996 14.8789,13.834 c 11.0957,10.0029 11.1543,27.3906 0.12695,37.4688 l -16.2949,15.6309 56.2559,66.4434 0.0742,0.0859 0.0664,0.0899 c 5.02734,6.53125 7.43164,11.5615 4.83984,17.9395 -3.13379,5.96875 -8.69727,10.29 -15.2559,11.8516 -0.67676,0.0908 -1.35938,0.13574 -2.04297,0.13671 l -0.004,-0.0117 c -4.42871,-0.27051 -8.61035,-2.13086 -11.7754,-5.24023 l -0.13086,-0.10743 -0.12304,-0.11132 -65.207,-59.127 -89.8965,86.2402 c 0,0 -5.33398,6.47754 -10.9707,6.61133 z m 178.104,-33.041 c 0.47852,-0.002 0.95703,-0.0332 1.43164,-0.0957 4.84277,-1.33301 8.95605,-4.54004 11.4316,-8.91016 1.44922,-3.5625 1.00293,-6.45703 -4.19727,-13.2148 l -56.0586,-66.2188 -26.375,25.3027 64.9551,58.9062 c 2.33887,2.41504 5.46484,3.91113 8.8125,4.2168 v 0.0137 z M 192.436,227.402 334.446,57.244 c 2.78711,-3.48926 4.5293,-6.97949 3.3418,-10.9121 -1.16113,-4.30273 -4.43457,-7.71777 -8.68555,-9.05664 -1.08105,-0.333984 -2.20508,-0.503906 -3.33594,-0.505859 -3.20801,0.07617 -6.26953,1.35645 -8.57617,3.58789 l -142.033,135.284999 c -11.1719,11.1719 -9.48242,26.0195 1.39258,36.8945 z" id="path2" style="stroke-width:0.416233"&gt;&lt;/path&gt;    &lt;/g&gt;  &lt;/g&gt;&lt;/svg&gt; 2024 &lt;br&gt; 
(Simon Tavaré Lab)
]

---

## Potential new directions
&lt;hr&gt;

- Can we analyze BCR- and RNA-seq data using RNN-VAE approach?

 &gt; Learn observed dynamics then make predictions for future (or past) timecourses
 
 &gt; Use dynamical models to guide
 
--

&lt;br&gt;

- Can we learn dynamics of one cell-state and then make predictions for another from its initial position?

 &gt; Between samples (Test and Control groups)

 &gt; From one infection to another


???
predicting cell states -- thats the whole point of VAE.
if we learn dynamics of one type can we predict of the other?
one infection to another?



---
count:false
class: center, middle

&lt;img src="figures/Thanks_slide.png" width="100%" style="display: block; margin: auto;" /&gt;


---
count:false

---
count:false

class: center, middle

We can identify B cell subsets during early stages of B cell response from published data.


&lt;img src="figures/community_umap.png" width="100%" style="display: block; margin: auto;" /&gt;


---

count:false
class: center, middle
Finding MZ B signature


&lt;img src="figures/umap_sig.png" width="100%" style="display: block; margin: auto;" /&gt;


---
count:false

### Division history of B cells
&lt;hr&gt;

- We used the **busulfan bone marrow** transplantation system to measure _total numbers_ and `\(f_d\)` in Fo and MZ B cells.


- In addition, we measured Ki67 expression &amp;rArr; extent of cell division in lymphocytes.




&lt;br&gt;

.pull-left[
- T and B cells continue expressing Ki67 protein for &amp;sim; 4 days after the division.


&gt; Residual expression from previous developmental stage? &lt;br&gt;
 &lt;br&gt;
&gt; Division in the current stage?

.citeR[
Verheijn _et al._ Cell Reports 2020
]
]

.pull-right[
&lt;img src="figures/div_hist.png" width="80%" style="display: block; margin: auto;" /&gt;


]

---
count:false

class: center
### Neutral model explains MZ B cell dynamics
&lt;hr&gt;

.middle[
&lt;img src="figures/MZ_fits2.png" width="85%" style="display: block; margin: auto;" /&gt;
]


---
count:false

class: center

### Time (Host age) dependence in FO B cell dynamics
&lt;hr&gt;

.middle[
&lt;img src="figures/FM_results2.png" width="85%" style="display: block; margin: auto;" /&gt;
]


---
count:false

### Insights into naive B cell homeostasis
&lt;hr&gt;


#### Naive B cells divide **very rarely.**
  - Inter-division times Fo B cells &amp;approx;  &gt; 1 year (CI: 0.6-23)
  - Inter-division times MZ B cells &amp;approx;  6 months (CI: 4-7)
  
  
--

#### Life-expectancy  of Fo B cells increases with mouse age.
  - Mean lifespan in a
    &gt; 10 wk old mouse &amp;approx;  35 days (CI: 2.5-4) &lt;/br&gt;
    &gt; 2 year old mouse &amp;approx;  61 days


#### Mean residence time of MZ B cells &amp;approx; 64 days (CI: 54-72) &lt;/br&gt;
    


#### Constitutive replenishment
  - Approximately ** 3% of Fo and  1.2% of MZ B pool** is replaced daily by source influx.
  

.citeR[
Unpublished data and Verheijn _et. al._ Cell Reports 2020
]

---
count:false

class: center, middle

### Fo B cell data from busulfan chimeras
&lt;hr&gt;

.middle[
&lt;img src="figures/FM_data.png" width="75%" style="display: block; margin: auto;" /&gt;
]


---
count:false
class: center, middle

### MZ B cell data from busulfan chimeras
&lt;hr&gt;

.middle[
&lt;img src="figures/MZ_data.png" width="75%" style="display: block; margin: auto;" /&gt;
]



    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false,
"nature": null,
"slideNumberFormat": "<div class=\"progress-bar-container\">\n  <div class=\"progress-bar\" style=\"width: calc(%current% / %total% * 100%);\">\n  </div>\n</div>"
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
// add `data-at-shortcutkeys` attribute to <body> to resolve conflicts with JAWS
// screen reader (see PR #262)
(function(d) {
  let res = {};
  d.querySelectorAll('.remark-help-content table tr').forEach(tr => {
    const t = tr.querySelector('td:nth-child(2)').innerText;
    tr.querySelectorAll('td:first-child .key').forEach(key => {
      const k = key.innerText;
      if (/^[a-z]$/.test(k)) res[k] = t;  // must be a single letter (key)
    });
  });
  d.body.setAttribute('data-at-shortcutkeys', JSON.stringify(res));
})(document);
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
